Python/pywarpx/picmi.py

```diff
@@ -422,6 +438,15 @@ def initialize_inputs(self):
         pywarpx.geometry.prob_lo = self.lower_bound  # physical domain
         pywarpx.geometry.prob_hi = self.upper_bound
 
+        # Handle PML
+        if ((self.bc_xmin == 'open') or (self.bc_xmax == 'open') \
+            or (self.bc_ymin == 'open') or (self.bc_ymax == 'open') \
```

Comment:
This is a picky detail, but the backslashes at the end of the lines are not needed (because of the enclosing parentheses). My preference is to not use them if not needed.